cfgmgr/teammgr.cpp

```diff
@@ -373,8 +375,33 @@ task_process_status TeamMgr::addLag(const string &alias, int min_links, bool fal
     string res;
 
     stringstream conf;
+
+    const string dump_path = "/var/warmboot/teamd/";
+    MacAddress mac_boot = m_mac;
+    /*set portchannel mac same with mac before warmStart ,when warmStart there is a file written by teamd*/
```

Comment:
Please change it to:
`/* set portchannel mac same with mac before warmStart, when warmStart there is a file written by teamd */`

Reply:
thanks,done